### PR TITLE
first recipe gatiab

### PR DESCRIPTION
**Pull request by Mustapha-hygeos**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by Mustapha-hygeos** [comment]
@conda-forge/help-python Hi, the recipe has passed all checks and is ready for you review. Thanks.

File: recipes/gatiab/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  sha256: 0fdd77decd846cff54f51542a1e370385ef8fe0936644a6c71ae95f2aed570e4
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9.19
```
Comment (synapticarbors) [inline recipes/gatiab/meta.yaml:19]: Please use the `python_min` convention for `noarch: python` packages shown in:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

**Review by synapticarbors — CHANGES_REQUESTED** [review]

File: recipes/gatiab/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - setuptools >=42
+    - pip
+  run:
+    - python {{ python_min }}
```
Comment (synapticarbors) [inline recipes/gatiab/meta.yaml:24]: This should be `- python >={{ python_min }}`

File: recipes/gatiab/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    - netcdf4 >=1.7.1
+    - scipy >=1.13.1
+    - tqdm >=4.66.5
+
+test:
+  imports:
+    - gatiab
+  commands:
+    - pip check
+  requires:
```
Comment (synapticarbors) [inline recipes/gatiab/meta.yaml:36]: Please add `- python {{ python_min }}` to the `test/requires` section.

**Review by synapticarbors — CHANGES_REQUESTED** [review]

**Pull request merged**